get-clients/readme.md

```diff
@@ -142,6 +139,43 @@ If your know the prospect directly or someone you know can introduce you to the
+
+> Be a Journalist. Investigate how they operate, what makes them good and where they are in need for help both before and during the phone call.
+
+Buy doing this the prospects experiences the quality of our journalistic skills, we gather valuable data on the subject and the market they are in and find out if we can be of actual service to them. Most importantly we are not being sales agents but journalists.
+
+### The stages of customer acquisition
+
+There are four stages a potential client goes trough and each stage can be an iteration. This way we can all contribute to the process of getting awesome clients.
+
+#### Potential client
+
+Companies that would make an awesome client. This includes a name.
+
+#### Prospect
+
+Companies that would make an awesome client with all the information we need to contact them. A prospect consists of the name of the company, who you can contact at the company, valuable information on the contact person, valuable information on the company and tangible examples of how our service can benefit the company and the contact person.
+
+#### Lead
+
+Prospects that have been contacted and need a follow up. Here you will find all information of the pervious stages and the notes of previous contact with the prospect and formulate the next step.
+
+#### Archive
+
+Leads that have been closed, either by a sale or disinterest. Here you find all information of previous stages and a not on how and why the lead has been archived.
+
+## Itterations on the four stages of customer acquisition.
+
+At new atoms we work with [achievables](.../user-stories-guide) to formulate our goals and manage our [workflow](.../board-guide).
+When you start working on a company in one of these stages it is key to move them to the next stage, just like you would with any other achievable. 
+
 ## The end
 
-There you have it, you officialy know more about direct customer aquisition than we do! One last note before you will start world domination: remember that you are a person with your own amazing traits and personality. Never put them aside in order to convice someone to hire your service. If you stay true to who you are, working with the people that hire you will be a lot more fun. You also don't have to pretend you're someone you're not, which is not as much fun as one would think.
+There you have it, you officially know more about direct customer acquisition than we do! One last note before you will start world domination: remember that you are a person with your own amazing traits and personality. Never put them aside in order to convince someone to hire your service. If you stay true to who you are, working with the people that hire you will be a lot more fun. You also don't have to pretend you're someone you're not, which is not as much fun as one would think.
```

Comment:
> There you have it, you officially know more about direct customer acquisition than we do!

don't get this... It is the sam right?

The rest of the conclusion is awesome though 👍